libs/designer/src/lib/core/index.ts

```diff
@@ -9,3 +9,4 @@ export { useNodeDisplayName } from './state/workflow/workflowSelectors';
 export { serializeWorkflow } from './actions/bjsworkflow/serializer';
 export { clearPanel, switchToWorkflowParameters } from './state/panel/panelSlice';
 export { useSelectedNodeId } from './state/panel/panelSelectors';
+export { initializeServices } from './state/designerOptions/designerOptionsSlice';
```

Comment:
this is just for convenience right? currently the host can import from `.../designerOptionsSlice` since it's already exported

Reply:
Good point, it looks like this is actually available:

```ts
import { initializeServices } from '@microsoft/logic-apps-designer/lib/core/state/designerOptions/designerOptionsSlice';
```

Should we just do that?

Reply:
I remember it not getting recognized and there being build errors when I tried using it without this change. Let me try again.

Reply:
This is the error I get when I try to import it without the change in LAUX:

Module not found: Can't resolve '@microsoft/logic-apps-designer/lib/core/state/designerOptions/designerOptionsSlice' in 'D:\power-platform-ux\packages\powerautomate-core\src\designer\v3Designer'

Reply:
![image](https://user-images.githubusercontent.com/8736762/229241539-7fd83625-e2be-4f14-bf38-3ef32dbf4506.png)

Reply:
It's not for convenience. It tells rollup and our build processes that it needs to be made available when it's bundled.

Reply:
Hm ok, so it seems like the typing shouldn't be generated then because the IDE doesn't complain

Reply:
I mean you can import types that way, it just wont work if you try to import js code that way. A lot of libraries export this way.